README.md

```diff
@@ -265,11 +265,11 @@ Yoti will provide a boolean result on the following checks:
 * Fraud list - Verify against  US Social Security Administration Fraud (SSN Fraud) list
 * Watch list - Verify against watch lists from the Office of Foreign Assets Control
 
-To use this functionality you must ensure your application is assigned to your Organisation in the Yoti Dashboard - please see [here](https://www.yoti.com/developers/documentation/#1-creating-an-organisation) for further information.
+To use this functionality you must ensure your application is assigned to your Organisation in the Yoti Hub - please see [here](https://www.yoti.com/developers/documentation/#1-creating-an-organisation) for further information.
```

Comment:
I was told the URI for the documentation should be changed to developers.yoti.com. So this should be:
https://developers.yoti.com/yoti-app-integration/yoti-app-integration#step-1-creating-an-organisation

Reply:
We have SDK-1081 to do that now. Since there's a small chance the links could change, I think it's best to leave this as is for now, and then we'll update the remaining ones in one go, once they've been confirmed 👍